app/models/mods_behaviors.rb

```diff
@@ -55,7 +55,7 @@ def to_solr(solr_doc = Hash.new, opts = {})
     solr_doc['series_ssim'] = gather_terms(self.find_by_terms(:series))
     #filter formats based upon whitelist
     solr_doc['resource_type_ssim'] = (gather_terms(self.find_by_terms(:resource_type)) & ['moving image', 'sound recording' ])
-    solr_doc['location_ssim'] = gather_terms(self.find_by_terms(:geographic_subject))
+    # solr_doc['location_ssim'] = gather_terms(self.find_by_terms(:geographic_subject))
```

Comment:
:+1: I like how this is left commented out.  In code it can historically document that some solr documents may have that field.